src/Apps/Artist/routes.tsx

```diff
@@ -226,6 +232,42 @@ export const routes: RouteConfig[] = [
       },
 
       // Routes not in tabs
+
+      {
+        path: "consign",
+        getComponent: () => ConsignRoute,
+        prepare: () => {
+          ConsignRoute.preload()
+        },
+        displayFullPage: true,
```

Comment:
I think this means that `found` will pass through any custom properties it doesn't recognize, for the route to access? Neat - I didn't know that.

Reply:
It doesn't _totally_ work out of the box, but one can perform a [special incantation](https://github.com/damassi/reaction/blob/f7dae56f3540fa3b20f10c66ad3553bcd6f3cf63/src/Artsy/Router/Utils/findCurrentRoute.tsx#L3) and pluck it out...